<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSongs</class>
 <widget class="QDialog" name="DlgSongs">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkauTan: Songs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="leSearch">
       <property name="placeholderText">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFilter">
       <property name="maxVisibleItems">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tblSongs">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblStats">
       <property name="text">
        <string notr="true">Total: 3456 songs (filtered out of 4567)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wLibraryRescan" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblLibraryRescan">
          <property name="text">
           <string>Library rescan:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="pbLibraryRescan">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddFile">
       <property name="text">
        <string>&amp;Add file...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddFolder">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Add &amp;folder...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>&amp;Remove...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRescanMetadata">
       <property name="text">
        <string>Re&amp;scan metadata</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddToPlaylist">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Add to &amp;playlist</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actRemoveFromLibrary">
   <property name="text">
    <string>&amp;Remove from library...</string>
   </property>
   <property name="toolTip">
    <string>Remove the songs from the library, but keep the files on the disk</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actDeleteFromDisk">
   <property name="text">
    <string>&amp;Delete from disk...</string>
   </property>
   <property name="toolTip">
    <string>Delete the songs from both the library and the disk</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actAddToPlaylist">
   <property name="text">
    <string>Add to playlist</string>
   </property>
   <property name="toolTip">
    <string>Add selected songs to playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actProperties">
   <property name="text">
    <string>Properties...</string>
   </property>
   <property name="toolTip">
    <string>Show song details in a dedicated dialog</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actRate">
   <property name="text">
    <string>Rate:</string>
   </property>
   <property name="toolTip">
    <string>Set the songs' local rating</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actTempoDetector">
   <property name="text">
    <string>Tempo detector...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actTapTempo">
   <property name="text">
    <string>Tap tempo...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actInsertIntoPlaylist">
   <property name="text">
    <string>Insert into playlist</string>
   </property>
   <property name="toolTip">
    <string>Insert the selected songs after the currently selected song in the playlist</string>
   </property>
  </action>
  <action name="actManualToId3">
   <property name="text">
    <string>Manual -&gt; ID3</string>
   </property>
   <property name="toolTip">
    <string>Move the Manual tag values into the file's ID3 tag</string>
   </property>
  </action>
  <action name="actFileNameToId3">
   <property name="text">
    <string>FileName -&gt; ID3</string>
   </property>
   <property name="toolTip">
    <string>Copy the FileName tag values into the file's ID3 tag</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>leSearch</tabstop>
  <tabstop>cbFilter</tabstop>
  <tabstop>tblSongs</tabstop>
  <tabstop>btnAddFile</tabstop>
  <tabstop>btnAddFolder</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnRescanMetadata</tabstop>
  <tabstop>btnAddToPlaylist</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
